src/sidebar/components/user-menu.js

```diff
@@ -43,6 +44,7 @@ import SvgIcon from '../../shared/components/svg-icon';
 function UserMenu({ auth, bridge, onLogout, serviceUrl, settings }) {
   const isThirdParty = isThirdPartyUser(auth.userid, settings.authDomain);
   const service = serviceConfig(settings);
+  const isFeatureEnabled = useStore(store => store.isFeatureEnabled);
```

Comment:
You need to use this like `const isNotebookEnabled = useStore(store => store.isFeatureEnabled('notebook_launch'))`, otherwise the `useStore` hook will never trigger a re-render (the `store.isFeatureEnabled` method reference never changes, and it is only when the result of the callback changes that a re-render happens.)

Reply:
🤦 Yep, that follows. Will fix.